include/items.php

```diff
@@ -2240,7 +2241,7 @@ function drop_item($id, $interactive = true) {
 		}
 
 		// If item has attachments, drop them
-		foreach (explode(", ", $item['attach']) as $attach) {
+		foreach (explode(",", $item['attach']) as $attach) {
```

Comment:
With the space after the ```,``` I would like it more. I guess this was done intentionally.